dream_layer_backend/create_test_images.py

```
@@ -0,0 +1,177 @@
+    # Create a new image with a random background color
+    colors = [(255, 200, 200), (200, 255, 200), (200, 200, 255), (255, 255, 200), (255, 200, 255), (200, 255, 255)]
+    bg_color = random.choice(colors)
+    
+    image = Image.new('RGB', (width, height), bg_color)
+    draw = ImageDraw.Draw(image)
+    
+    # Try to use a default font, fallback to basic if not available
+    try:
+        font_size = 24
+        font = ImageFont.truetype("/System/Library/Fonts/Arial.ttf", font_size)
+    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)